lib/trino-filesystem-gcs/src/main/java/io/trino/filesystem/gcs/GcsStorageFactory.java

```diff
@@ -113,6 +117,7 @@ public Storage create(ConnectorIdentity identity)
                             .setInitialRetryDelay(minBackoffDelay)
                             .setMaxRetryDelay(maxBackoffDelay)
                             .build())
+                    .setHeaderProvider(() -> Map.of(USER_AGENT, StorageOptions.getLibraryName() + "/" + StorageOptions.version() + " " + applicationId))
```

Comment:
Is there a way to structure that so we do not have to know that default user agant was `StorageOptions.getLibraryName() + "/" + StorageOptions.version()`. In case default value would change in the future.

Reply:
I couldn't find any other way